api/server/cluster.go

```diff
@@ -346,6 +292,18 @@ func (c *clusterApi) nodeHealth(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(nodeOkMsg + "\n"))
 }
 
+// swagger:operation GET /cluster/peerstatus node peerstatus peerStatus
+//
+// This will return the peer node status
+//
+// ---
+// produces:
+// - application/json
+// responses:
+//   '200':
+//      description: anode
+//      schema:
+//         type: string
```

Comment:
this API takes in a nodeID as an argument

Reply:
it does? i didn't see it in the route. but good catch!